### PR TITLE
Support value_counts and plot in DC

### PR DESCRIPTION
**Pull request by shiyu22**
Signed-off-by: shiyu22 <shiyu.chen@zilliz.com>

### CODE REVIEW

**Comment by shiyu22** [comment]
#950

File: towhee/functional/mixins/display.py [hunk 1]
```diff
@@ -11,21 +11,75 @@
...
+    return wrapper
+
+
+def _plot_callback(self):
+    def wrapper(_: str, index: Tuple[str], *arg, **kws):
+        # pylint: disable=import-outside-toplevel
+        # pylint: disable=unused-argument
+        import pandas as pd
+        columns = [[] for _ in index]
+        for x in self:
```
Comment (filip-halt) [inline towhee/functional/mixins/display.py:45]: wont both of these callbacks consume the data of a streamed iterator? Will value_counts and plot only be supported as the last function of a DC chain like to_list()

Comment (shiyu22) [reply towhee/functional/mixins/display.py:45]: Yes, it will. Maybe I will add a check(transform to upstream) for it.

**Comment by shiyu22** [comment]
Another PR #968

File: towhee/functional/mixins/display.py (hunk 1 above)
Comment (shiyu22) [reply towhee/functional/mixins/display.py:45]: We call this func with `param_scope().callholder`, so it will run in the workflow as usual functions.

**Pull request closed without merging**